File: challenges/4-emprestimo/typescript/clintonrocha/emprestimo.ts
import * as readline from "readline"

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
  });
  rl.question("number: ", (input) => {
    const valor_emprestimo = parseFloat(input.split(",")[0]);
    const taxa_juros = parseInt(input.split(",")[1]);
    const tempo = parseInt(input.split(",")[2]);
    const valor_final = valor_emprestimo * (1 + taxa_juros / 100) ** tempo;
    console.log(valor_emprestimo, taxa_juros, tempo)
    console.log(valor_final.toFixed(2));
    rl.close();
  });

File: challenges/4-emprestimo/python/davidspader/emprestimo.py
from sys import argv

valores = argv[1].split(',')

valor_emprestimo = float(valores[0])
taxa_juros = int(valores[1])
tempo = int(valores[2])

valor_final = valor_emprestimo * (1 + taxa_juros/100)**tempo

print("{:.2f}".format(valor_final))